src/Components/CameraFeed/ConfigureCamera.tsx

```
@@ -419,6 +551,233 @@ export default function ConfigureCamera(props: Props) {
+                                    value={
+                                      boundary[direction]?.toString() ?? ""
+                                    }
+                                    className="mt-1"
+                                    onChange={(e) =>
+                                      setBoundary((prev) => ({
+                                        ...prev,
+                                        [direction]: Number(e.value),
+                                      }))
+                                    }
+                                    errorClassName="hidden"
+                                  />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)